src/frontend/js/modules/search/form/advanced/search-advanced-form.controller.js

```diff
@@ -53,6 +53,18 @@ require('../../query/search-query.service.js');
     function loadProviders() {
       esnSearchContextService.getProvidersContext().then(function(providers) {
         self.providers = providers;
+
+        self.selectedProvider = undefined;
+
+        var activeProviders = (self.providers || []).filter(function(provider) {
+          return provider.active;
+        });
+
+        if (activeProviders.length) {
+          self.selectedProvider = activeProviders[0];
+        }
+
+        onProviderSelected(self.selectedProvider);
```

Comment:
Don't forget to write test for these changes

Reply:
This is a transition PR to remove the Global search, let's do this in the next PR

Reply:
> This is a transition PR to remove the Global search, let's do this in the next PR

That is not the reason. PR should really `complete` and you should write for it, if it is a `transition PR`, please mark it as a draft PR and finish it before merging.

Take time to save time in the future.

Reply:
I didn't have time to manage this because I'm in half on Inbox and half on SmartSla, currently there is no test for search so we can stay like that for the moment and someone with more time working on the search will handle it.

Reply:
At least you should add a comment or create an issue for this.

```
// TODO: Write tests for this (https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/issues/110)
```